tests/Microsoft.Data.Analysis.Tests/DataFrameTests.cs

```diff
@@ -2136,5 +2159,133 @@ public void TestApply()
                 Assert.Equal(newCol[i], values[i] + 0.5d);
             }
         }
+
+        private void RollingColumnWindowVerifyElementwiseEquals<T>(PrimitiveDataFrameColumn<T> verify, PrimitiveDataFrameColumn<T> values)
```

Comment:
This isn't really "RollingColumnWindow" specific, is it? Can't this just be "VerifyElementwiseEquals(PrimitiveDataFrameColumn<T> expected, PrimitiveDataFrameColumn<T> actual)`?